cliquet/views/errors.py

```diff
@@ -82,6 +82,10 @@ def error(context, request):
         logger.critical(context.original, exc_info=True)
         return service_unavailable(context, request)
 
+    if hasattr(request, 'raven'):
+        ident = request.raven.get_ident(request.raven.captureException())
+        print("Exception captured in event number: %s" % ident)
```

Comment:
Actually we catch everything that is raised in a view, since it is a webserver we should catch everything that is not related to server startup.